src/shaders/hillshade_prepare.fragment.glsl

```diff
@@ -41,7 +41,7 @@ void main() {
     float b = getElevation(v_pos + vec2(0, -epsilon.y));
     float c = getElevation(v_pos + vec2(epsilon.x, -epsilon.y));
     float d = getElevation(v_pos + vec2(-epsilon.x, 0));
-    float e = getElevation(v_pos);
+    // float e = getElevation(v_pos);
```

Comment:
Not sure if I should delete this line or leave it as a comment, in case this data is needed in the future.

Reply:
Fair point, updated :)